### PR TITLE
Connector for genie.co.kr

### PR DESCRIPTION
**Pull request by nerogit**
http://genie.co.kr is one of famous Korean music streaming service

### CODE REVIEW

**Comment by nerogit** [comment]
Thanks for advice:) and I modified it

File: connectors/v2/genie.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+Connector.playerSelector = '#music-player';
+Connector.artistSelector = '#ArtistNameArea';
+Connector.trackSelector = '#SongTitleArea';
+Connector.albumSelector = '#AlbumTilteArea';
+Connector.currentTimeSelector = '#playTime span';
+Connector.durationSelector = '#playTime strong';
+
+Connector.isPlaying = function() {
+	var btn = $('#PlayBtnArea')[0];
```
Comment (inverse) [inline connectors/v2/genie.js]: I would simplify this to 

```
Connector.isPlaying = function () {
	return $('#PlayBtnArea').hasClass('active');
};
```

Or is there really more than one instance of `#PlayBtnArea`?

Comment (inverse) [inline connectors/v2/genie.js]: @alexesprit I hadn't visited the site - but I was more commenting on the use of jquery to simplify the function.

Comment (alexesprit) [inline connectors/v2/genie.js]: @inverse got it.

Comment (nerogit) [inline connectors/v2/genie.js]: It's just for using `classList` property. I changed it to use `hasClass` method. Thanks @inverse

**Comment by alexesprit** [comment]
Works fine! 👍

**Comment by inverse** [comment]
Thanks for doing this :+1:

**Pull request merged**